### DISCUSSION TITLE
Connect without SSL

### DISCUSSION THREAD

**Discussion by Keda87** [planetscale/discussion#143] [Q&A] [answered]
Hi, I'm testing planetscale for my development using golang.
I'm trying to connect to my database, but I got this error.
```
panic: Error 1105: unknown error: Code: UNAVAILABLE
server does not allow insecure connections, client must use SSL/TLS
```
is it possible for me to skip the SSL?
and here is my DSN
```
func (conf DBConfig) ToDSN() string {
	return fmt.Sprintf("%s:%s@tcp(%s:%s)/%s", conf.DBUser, conf.DBPass, conf.DBHost, conf.DBPort, conf.DBName)
}
```

**Comment by jgreet** [comment, marked as answer]
Hi @Keda87! It isn't possible to disable SSL. We'd prefer to never have PlanetScale database credentials traveling over the public internet unencrypted.

Can you try adding `?tls=true` to the end of that connection string to [enable TLS/SSL](https://github.com/go-sql-driver/mysql#tls)?

**Reply by Keda87** [reply]
@jgreet thank you to point it out, it works now by adding `?tls=true`

**Reply by NLanese** [reply]
Im having the same issue and adding `?tls=true` did nothing

**Reply by fooforge** [reply]
I've responded to this in https://github.com/planetscale/discussion/discussions/354#discussioncomment-4422805.

**Reply by ajaythxkur** [reply]
How to do that with node js sequelize library?
